### PR TITLE
fix: android onboarding remove corousel

### PR DESCRIPTION
**Pull request by Dmitry-Borodin**
Actually decided to remove onboarding coousel
and
 fix: android onboarding last step second box was not shown in real pager

### CODE REVIEW

**Review by prybalko — APPROVED** [review]

File: android/src/main/java/io/parity/signer/ui/rootnavigation/RootNavigationGraph.kt [hunk 1]
```diff
@@ -81,23 +81,16 @@ fun NavGraphBuilder.firstTimeOnlyOnboarding(
...
-		}
 		composable(route = FirstTimeOnboarding.termsConsentRoute) {
 			TermsConsentScreenFull(navigateNextScreen = onNextStepsNavigate)
 		}
 	}
 }
 
 private object FirstTimeOnboarding {
-	const val onboardingExplanationRoute = "navigation_onboarding_explanation"
+//	const val onboardingExplanationRoute = "navigation_onboarding_explanation"
```
Comment (krodak) [inline android/src/main/java/io/parity/signer/ui/rootnavigation/RootNavigationGraph.kt:93]: Shouldn't this be removed, given we have git history to bring it back?

File: android/src/main/java/io/parity/signer/screens/initial/explanation/OnboardingScreen4.kt [hunk 2]
```diff
@@ -26,24 +26,26 @@ import io.parity.signer.ui.theme.*
...
-		contentAlignment = Alignment.BottomCenter,
-	) {
-		PrimaryButtonWide(
-			label = stringResource(R.string.onboarding_cta_continue),
-			activeBackground = Color.White,
-			activeText = MaterialTheme.colors.pink500,
-			modifier = Modifier.padding(24.dp),
-			onClicked = onContinue,
+	Box() {
+		OnboardingScreenGeneric(
```
Comment (krodak) [inline android/src/main/java/io/parity/signer/screens/initial/explanation/OnboardingScreen4.kt:30]: Shouldn't all onboarding screens be removed in this PR? We can always bring them back from git history

**Review by krodak — COMMENTED** [review]
If we don't plan on bringing back this onboarding very soon, I think deleting all screens and related resources could be a good move, I followed that within #2104

File: android/src/main/java/io/parity/signer/ui/rootnavigation/RootNavigationGraph.kt (hunk 1 above)
Comment (Dmitry-Borodin) [reply android/src/main/java/io/parity/signer/ui/rootnavigation/RootNavigationGraph.kt:93]: will remove on subsequent redesign pr

File: android/src/main/java/io/parity/signer/screens/initial/explanation/OnboardingScreen4.kt (hunk 2 above)
Comment (Dmitry-Borodin) [reply android/src/main/java/io/parity/signer/screens/initial/explanation/OnboardingScreen4.kt:30]: I have a feeling we will need it soon -) If not will remove soon, but it's much easier to search in existing files than in a history for unknown filenames

**Pull request merged**